topics/getting-started.md

```diff
@@ -1,7 +1,15 @@
 [//]: # (title: Getting started)
 [//]: # (auxiliary-id: Getting_started)
 
-There are different ways to get started with Kotlin Multiplatform Mobile (KMM) depending on whether you want to start from scratch or in an existing Android application.
+## What is KMM?
+
+Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Developer Advocate Ekaterina 
+Petrova will explain what Kotlin Multiplatform Mobile is and how you can use it in your project. Together with Ekaterina 
+you'll set up an environment and prepare for creating your first cross-platform mobile application with KMM.
+
+<video width="560" height="315" href="mdN6P6RI__k" title="Kotlin Multiplatform Multiverse, Episode 1: Meet KMM!"/>
+
+After watching the video you can continue creating first KMM application or make a sample Android application work well on iOS.
```

Comment:
I like the overall description but I think that this sentence is not necessary. It doesn't add any value and it is not clear how it's connected with `make a sample Android app work well on iOS`.
This page is short and simple and easy to navigate. Users will see other headings.